android/src/main/java/com/iterable/reactnative/RNIterableAPIModule.java

```diff
@@ -101,7 +101,7 @@ public void updateEmail(String email) {
     @ReactMethod
     public void updateEmail(String email, @Nullable String token) {
         IterableLogger.d(TAG, "updateEmail: " + email + ", token: " + token);
-        IterableApi.getInstance().updateEmail(email, token);
+//        IterableApi.getInstance().updateEmail(email, token);
```

Comment:
Why is this commented out?

Reply:
the authToken methods are private.. and not accessible from here. Have separate PR which comments the entire functions 😅